src/TypeSpec.Extension/Emitter.Csharp/test/Unit/model-type.test.ts

```diff
@@ -556,34 +540,18 @@ op op5(@body body: IsFooArray): IsFooArray;
         assert(isFooModel !== undefined);
         assert(isFooArrayModel !== undefined);
         // assert the inherited dictionary type is expected
+        strictEqual(isUnknownModel.AdditionalProperties?.Kind, "any");
+
+        strictEqual(isStringModel.AdditionalProperties?.Kind, "string");
+
+        strictEqual(isInt32Model.AdditionalProperties?.Kind, "int32");
+
+        deepStrictEqual(isFooModel.AdditionalProperties, fooModel);
+
+        strictEqual(isFooArrayModel.AdditionalProperties?.Kind, "array");
         deepStrictEqual(
-            {
-                Kind: "any",
-                Encode: undefined
-            },
-            isUnknownModel.AdditionalProperties
-        );
-        deepStrictEqual(
-            {
-                Kind: "string",
-                Encode: undefined
-            },
-            isStringModel.AdditionalProperties
-        );
-        deepStrictEqual(
-            {
-                Kind: "int32",
-                Encode: undefined
-            },
-            isInt32Model.AdditionalProperties
-        );
-        deepStrictEqual(fooModel, isFooModel.AdditionalProperties);
-        deepStrictEqual(
-            {
-                Kind: "array",
-                ValueType: fooModel
-            },
-            isFooArrayModel.AdditionalProperties
+            isFooArrayModel.AdditionalProperties.ValueType,
+            fooModel
```

Comment:
I changed these because of two reasons:
1. we are using the `strictEqual` and `deepStrictEqual` wrong, the `actual` is the first parameter but we are using it as `expected`.
2. we should be focusing on the properties we need to validate by this test case, instead of just simply asserting the model is the same which is hard to read and hard to understand.